static/js/reducers/ui_test.js

```diff
@@ -266,6 +270,48 @@ describe('ui reducers', () => {
   });
 
   describe('Enrollment', () => {
+    describe('enrollment message', () => {
+      it('should have a null default value', () => {
+        assert.equal(store.getState().ui.enrollMessage, null);
+      });
+
+      it('should let you set the message', () => {
+        return dispatchThen(setEnrollMessage("value"), [
+          SET_ENROLL_MESSAGE
+        ]).then(state => {
+          assert.equal(state.enrollMessage, "value");
+        });
+      });
+    });
```

Comment:
we're using this exact same test pattern a few dozen times in this file now. would be nice to have a helper method

```javascript
export const assertReducerResultState = (action, actionType, stateParamName, value) => {
  return dispatchThen(action(value), [actionType]).then(state => {
    assert.equal(state[stateParamName], value, `$(action) should set state.$(stateParamName) to $(value)`);
  });
};

assertReducerResultState( setEnrollMessage, SET_ENROLL_MESSAGE, 'enrollMessage', 'new_value' );
```

readability with 4 arguments isn't the greatest, but at this point i think it's well worth it. your default value check could also be implemented with a helper method but that's just 1 line anyway - you can take or leave that one